Improve the map documentation (#107)
hazelcast/imdg-docs: docs/modules/data-structures/pages/backing-up-maps.adoc

= Backing Up Maps

> :description: Backing up maps helps to protect your data in case a cluster member goes down. For each backup, Hazelcast creates a copy of the map and distributes it across another member. Because each backup is a copy, the number of backups in your cluster affects the amount of memory that your members need.

Author: ```suggestion
:description: Backing up maps helps to protect your data in case a cluster member goes down. For each backup, Hazelcast creates a copy of the map entries and distributes them to another member. Because each backup is a copy, the number of backups in your cluster affects the amount of memory that your members need.
```

> = Making My Data Safe

Author: I'd title this 'Making your Map Data Safe'.

We don't use first person in the docs.
